samples/basic/minimal/sample.yaml

```diff
@@ -105,3 +105,15 @@ tests:
       - kernel
     integration_platforms:
       - qemu_riscv32
+  sample.minimal.no-mt-no-sw-isr-table.arm:
+    extra_args: EXTRA_CONF_FILE='common.conf;no-mt.conf;no-sw-isr-table.conf;arm.conf'
```

Comment:
line related to comment: https://github.com/zephyrproject-rtos/zephyr/pull/94132#discussion_r2410028659